helpers/discordAuth.ts

```diff
@@ -0,0 +1,76 @@
+import { prisma } from '../prisma'
+const qs = require('qs')
```

Comment:
Why not simple import? Also, do we really need a new library for this? As far as I understand `qs` helps with deeply nested string parsing.

Reply:
ok, i replaced it with URLSearchParams like they did here:
https://discordjs.guide/oauth2/#authorization-code-grant-flow